src/pspm_pfm.m

```diff
@@ -0,0 +1,588 @@
+end
+
+% Checking baseline
+if ~isfield(model,'baseline')
+    model.baseline = 0;
+elseif ~isnumeric(model.baseline)
+    warning('ID:invalid_input','''model.baseline'' has to be a numeric.'); return;
+elseif model.baseline > model.window || model.baseline < 0
+    warning('ID:invalid_input',['''model.baseline'' has to be positive ',...
+                                'and smaller than ''model.window''.']); return;
+end
+
+% Checking marker channels 
+if strcmpi(model.timeunits,'markers')
+    if ~isfield(model,'marker_chan')
+        model.marker_chan = 'marker';
+    elseif numel(model.marker_chan)==1 && ~isnumeric(model.marker_chan)
+        warning('ID:invalid_input','Marker channels have to be numeric.'); return;
+    elseif numel(model.marker_chan)>1 && ...
+           ( ~iscell(model.marker_chan) || ... 
+             numel(model.marker_chan)~=numel(model.marker_chan) )
+        warning('ID:invalid_input',['Marker channels have to be a cell array', ...
+                                    ' of the same size than ''model.modelfile''.'])
+    end
+end
+
+% Checking standard experimental condition
+std_cond_war_msg = ['The standard experimetal condition must be',...
+                ' either a valid experimental condition or an',...
+                ' index corresponding to it.'];
+if ~isfield(model,'std_exp_cond')
+    model.std_exp_cond = 'none';
+elseif ~ischar(model.std_exp_cond) && ~isnumeric(model.std_exp_cond)
+    warning('ID:invalid_input',std_cond_war_msg); return;
+elseif ischar(model.std_exp_cond)
+    tmp_ind = cellfun(@(x) strcmpi(model.std_exp_cond,x),model.timing{1}.names);
+    if ~any(tmp_ind)
+        warning('ID:invalid_input',std_cond_war_msg); return;
+    end
+    std_exp_cond.name = model.std_exp_cond;
+    std_exp_cond.ind = find(tmp_ind);
+elseif isnumeric(model.std_exp_cond)
+    if model.std_exp_cond < 1 || ...
+       model.std_exp_cond > numel(model.timing{1}.names)
+        warning('ID:invalid_input',std_cond_war_msg); return;
+    end
+    std_exp_cond.name = model.timing{1}.names(model.std_exp_cond);
+    std_exp_cond.ind = model.std_exp_cond;
+end
+clear std_cond_war_msg tmp_ind
+
+%Checking norm
+if ~isfield(model, 'norm')
+    model.norm = 0;
+elseif ~ismember(model.zscore, [0, 1])
+    warning('ID:invalid_input', '''model.zscore'' has to be 0 or 1.'); return;
+end
+
+%%% Checking options %%%
+if ~isfield(options, 'overwrite')
+    options.overwrite = 0;
+elseif ~ismember(options.overwrite, [0, 1])
+    options.overwrite = 0;
+end
+
+if exist(model.modelfile, 'file') && ~(isfield(options, 'overwrite') && options.overwrite == 1)
+    overwrite=menu(sprintf('Model file (%s) already exists. Overwrite?', model.modelfile), 'yes', 'no');
+    if overwrite == 2, return, end
+    options.overwrite = 1;
+end
+
+%%
+%%%%%%%% Loading files %%%%%%%%
+
+fprintf('Computing Pupil Model: %s \n', model.modelfile);
+
+nExpCond = numel(model.timing{1}.names);     % number of experimental conditions
```

Comment:
(Low priority) This is not a big deal, so you can ignore it if you want, but for completeness' sake we should stick to a single variable naming convention. I know this is not the case for PsPM overall, but at least for new files we can try to use underscore naming, i.e. `nExpCond` --> `n_exp_cond` and so on.

Reply:
Yeah I know that its just that it is the convention in `pspm_glm` (I mean it is the same variable name) so I found that I'll not change it. But okay will do it  ;)